### DISCUSSION TITLE
Adding an enclosure : page not found !

### DISCUSSION THREAD

**Discussion by StanMOI** [sanni/cartreader#110] [Need help]
Link are not available anymore !
https://github.com/sanni/cartreader/Adding-an-enclosure
Please help.

**Comment by sanni** [comment]
Thanks for letting me know, seems like quite a few links got broken in the Wiki, should be updated now.

**Reply by StanMOI** [reply]
Still not working :=)

**Reply by sanni** [reply]
The correct link is: https://github.com/sanni/cartreader/wiki/Adding-an-enclosure
Where did you find the wrong link so I can replace it?

**Reply by StanMOI** [reply]
Sorry, dont remember where :))
